cluster/manifests/skipper/rbac.yaml

```diff
@@ -45,6 +60,51 @@ rules:
   - list
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: skipper-ingress-routesrv
+rules:
+- apiGroups:
+  - networking.k8s.io
+  resources:
+  - ingresses
+  verbs:
+  - get
+  - list
+- apiGroups: [""]
```

Comment:
I see this permission **not copied** from skipper-ingress ClusterRole. Is it okay to drop it?

```
- apiGroups: ["extensions"]
  resources: ["ingresses", ]
  verbs: ["get", "list"]
  ```

Reply:
That's the old group for ingress v1beta1 which we dropped more than a year ago.